sdk/src/contexts/solana/utils/computeBudget/index.ts

```diff
@@ -89,14 +97,20 @@ export async function determineComputeBudget(
   ];
 }
 
+const DEFAULT_PRIORITY_FEE = (_: PublicKey[]) => Promise.resolve(1); // Set fee to 1 microlamport by default
+
 async function determinePriorityFee(
   connection: Connection,
   lockedWritableAccounts: PublicKey[] = [],
   percentile: number,
+  customDeterminePriorityFee: (
```

Comment:
I don't see how it would be useful to an integrator because we're not providing the `connection` to this custom function, only the addresses. they can't do much with only addresses unless they're providing their own connection.

also, the way you're using it here is you're just setting the default value using it, rather than the final value. it will still get overridden by our own logic, which follows it in this function starting on line 115. so this doesn't seem like it will even work.

It seems to me we want to either:
- let integrators completely override the `determinePriorityFee` function
- or let them just provide a custom percentile and keep our logic the same
 
this code change is sort of a weird combination of the two.

I would keep things simple and go with option 2 for now. can we just remove this custom callback function, and only add the percentile `number` as a new config option?

if there's need for deeper customizability in the future we can revisit overriding `determineComputeBudget` later.

Reply:
So this solution wont' work? I asked @barnjamin about this functionality but i'm having problems trying to clone this commit and installing it to my react app so i couldn't test it, and as i see it didn't merged to development brach yet right?

Reply:
good call. I’ll add it as a parameter. The idea is to allow integrators to use a different API, like what Helius provides for estimating priority fees, or any custom solution that someone might want to try, the limit is the skies 😅

Reply:
I think it would be good to offer both: custom percentile, or custom `determinePriorityFee` function altogether.